### PR TITLE
Update README with ssh key information

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
cc @aravindhp @akhil-rane @openshift/openshift-team-windows-containers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,8 +19,11 @@ To run the e2e tests for WMCO locally against an OpenShift cluster set up on AWS
 export KUBECONFIG=<path to kubeconfig>
 export AWS_SHARED_CREDENTIALS_FILE=<path to aws credentials file>
 export CLUSTER_ADDR=<cluster_name, eg: ravig211.devcluster.openshift.com>
-export KUBE_SSH_KEY_PATH=<path to ssh key>
+export KUBE_SSH_KEY_PATH=<path to ssh key>
```
Comment (ravisantoshgudimetla) [inline README.md]: extra space. Will remove it.

File: README.md [hunk 2]
```diff
@@ -19,8 +19,11 @@ To run the e2e tests for WMCO locally against an OpenShift cluster set up on AWS
 export KUBECONFIG=<path to kubeconfig>
 export AWS_SHARED_CREDENTIALS_FILE=<path to aws credentials file>
 export CLUSTER_ADDR=<cluster_name, eg: ravig211.devcluster.openshift.com>
-export KUBE_SSH_KEY_PATH=<path to ssh key>
+export KUBE_SSH_KEY_PATH=<path to ssh key> 
 ```
+Make sure that the key you're using in KUBE_SSH_KEY_PATH is 
+private key part of the keypair mentioned at [sshKeyPair](https://github.com/openshift/windows-machine-config-operator/blob/42593c5d2eb798b572c58b5debafc4c392d1f967/test/e2e/wmco_test.go#L59) within the instance CR
```
Comment (aravindhp) [inline README.md]: This implies that you need to change `KUBE_SSH_KEY_PATH` to point to `libra` which we cannot do in development. This should rather say, `Update the keypair mentioned at sshKeyPair to match what is being used in KUBE_SSH_KEY_PATH`. Also call out that `libra` is being used for CI purposes.

BTW, did you try hard coding to `openshift-dev`? I think the test platform team has uploaded that key to all AWS regions in the CI account when they decided to do region sharding for CI. Please try this out in a separate PR and move to using that if it works.

Comment (ravisantoshgudimetla) [reply README.md]: > BTW, did you try hard coding to openshift-dev? I think the test platform team has uploaded that key to all AWS regions in the CI account when they decided to do region sharding for CI. Please try this out in a separate PR and move to using that if it works.

I tried it multiple times in the CI for #15 and it failed.

> This implies that you need to change KUBE_SSH_KEY_PATH to point to libra which we cannot do in development. This should rather say, Update the keypair mentioned at sshKeyPair to match what is being used in KUBE_SSH_KEY_PATH. Also call out that libra is being used for CI purposes.

Will do that.

**Comment by aravindhp** [comment]
/hold

Until #23 merges

**Comment by ravisantoshgudimetla** [comment]
/hold cancel

as #23 merged

**Pull request merged**